src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Helpers/Conversions/RecoveryPointConversions.cs

```diff
@@ -44,7 +44,9 @@ public static List<RecoveryPointBase> GetPSAzureRecoveryPoints(ServiceClientMode
             List<RecoveryPointBase> result = new List<RecoveryPointBase>();
             foreach (ServiceClientModel.RecoveryPointResource rp in rpList.RecoveryPointList.RecoveryPoints)
             {
-                ServiceClientModel.RecoveryPoint recPoint = rp.Properties as ServiceClientModel.RecoveryPoint;
+                if (rp.Properties.GetType() == typeof(ServiceClientModel.RecoveryPoint))
+                {
+                    ServiceClientModel.RecoveryPoint recPoint = rp.Properties as ServiceClientModel.RecoveryPoint;
 
                 DateTime recPointTime = DateTime.ParseExact(recPoint.RecoveryPointTime, @"MM/dd/yyyy HH:mm:ss", CultureInfo.InvariantCulture);
```

Comment:
doesn't seem to be formatted properly. Can you try doing a Ctrl+K,D to autoformat?